extensions/machine-learning-services/src/common/utils.ts

```diff
@@ -176,3 +182,48 @@ export async function promptConfirm(message: string, apiWrapper: ApiWrapper): Pr
 
 	return choices[result.label] || false;
 }
+
+export function makeLinuxPath(filePath: string): string {
+	const parts = filePath.split('\\');
+	return parts.join('/');
+}
+
+/**
+ *
+ * @param currentDb Wraps the given script with database switch scripts
+ * @param databaseName
+ * @param script
+ */
+export function withDbChange(currentDb: string, databaseName: string, script: string): string {
```

Comment:
nit: not important but the function name seems tad bit confusng with not verb :) .. should be something like getDBChangeScript or getScriptWithDBChange or getUseDBscript... somcething like that may be?